tools/unwrapfile/unwrapfile.cc

```diff
@@ -181,76 +176,111 @@ void unwrapMP4Frames(const char *filepathRaw, unsigned int frameInterval, const
 }
 }   // Anonymous namespace
 
+void usage()
+{
+    std::cout << "usage: unwrapfile [--no-sound] [--extract-frames[=SKIP]] [--camera CAMERA_NAME] [--unwrapped-resolution WIDTHxHEIGHT] [file(s)]" << std::endl;
+}
+
 int main(int argc, char** argv)
 {
     if (argc == 1) {
```

Comment:
For the CLI tools I made for the ALife paper we used a really nice CLI library (https://github.com/dannyneil/spiking_relu_conversion/blob/master/normalize_cnn_data.m#L8) which was much more modern c++ and terser - would you mind horrible switching to that (and adding it to third_party)

Reply:
Done. That is a much nicer library!

The only issue I had is that I couldn't find a way to get the remaining args, which in our case are filenames, so I had to add a ``-i`` option for that instead. Any ideas? There is an ``app.remaining()`` method, which seems to do that, but when I ran the program I was getting an "unrecognised argument" error.

Reply:
I *think* you need to call ``app.allow_extras(true)`` first and *then* ``app.remaining()`` *should* work...

Reply:
That seemed to do the trick. Thanks!